src/Compilers/CSharp/Portable/BoundTree/Statement.cs

```diff
@@ -591,6 +592,24 @@ internal partial class BoundBadStatement : IInvalidStatement
     {
         protected override OperationKind StatementKind => OperationKind.InvalidStatement;
 
+        ImmutableArray<IOperation> IInvalidStatement.Children
+        {
+            get
+            {
+                var builder = ArrayBuilder<IOperation>.GetInstance(this.ChildBoundNodes.Length);
+                foreach (var childNode in this.ChildBoundNodes)
+                {
+                    var operation = childNode as IOperation;
+                    if (operation != null)
```

Comment:
It looks like `BoundExpression` implements `IOperation` so this should not fail.